website/hw/hw3_rubric.rst

```diff
@@ -0,0 +1,45 @@
+Homework 3 Rubric
+=================
+
+The grading of this project follows a specifications grading approach. If you have not already
+done so, make sure to read our `Grading <../grading.html>`__ page for more details.
+
+When grading your work, the graders will be checking for the following issues in your code. Please
+note that some of the issues are labelled as "[Major]" (these will have a greater effect on your score
+for the homework)
+
+Writing Makefiles + Building a library
+--------------------------------------
+
+- [Major] Not adding a ``Makefile`` in ``polygon-area``
+- Adding the final (complete) ``Makefile`` in a single commit. You should try
+  to make a few intermediate commits as you create the ``Makefile`` described
+  in this section.
+
+Task 1: Makefile for the micro editor
+-------------------------------------
+
+- [Major] Not adding a ``Makefile`` in ``micro-editor/``
+- Having individual rules for each object file
+- Not using variables (``CC``, ``CFLAGS``, etc.)
+- Makefile does not correctly produce a ``micro`` executable.
+
+Task 2: Makefile for the micro editor
```

Comment:
I think this is meant to say Makefile for the librgraph library

Reply:
Fixed